### DISCUSSION TITLE
whay im not eligible ?

### DISCUSSION THREAD

**Discussion by Alirs98** [zkSync-Community-Hub/zksync-developers#583] [General]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

hi zksync team 
its my wallet 0x5894Bad5398eD2A53f4839a59A61c62fBef529eA 
what im not eligible ?!
This is not logical at all. I have the conditions you mentioned, but I am not eligible

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.
